tests/test_datamodel_service.py

```diff
@@ -833,3 +834,21 @@ def test_dynamic_dependency(new_meshing_session):
     d = ic.Refaceting.Deviation.get_state()
     cd = ic.Refaceting.CustomDeviation.get_state()
     assert d == cd
+
+
+def test_field_level_help(new_meshing_session, capsys):
+    meshing = new_meshing_session
+    import_geometry = meshing.meshing.ImportGeometry.create_instance()
+    assert isinstance(import_geometry.FileFormat, PyCommandArgumentsSubItem)
+    import_geometry.FileFormat.help()
```

Comment:
@prmukherj Can you please check if this function is mapped to the workflow level?

Reply:
It is there at the task object level but not individual commands.

Reply:
I think "helpString" is not present at this level?

Reply:
Is it because `watertight.import_geometry.file_format.help()` doesn't call `PyCommandArgumentsSubItem.help`?